packages/react/src/components/Table/TableBody/TableBody.jsx

```diff
@@ -1,16 +1,18 @@
 import React, { useMemo } from 'react';
 import PropTypes from 'prop-types';
 import { TableBody as CarbonTableBody } from 'carbon-components-react';
+import classNames from 'classnames';
```

Comment:
Since this gets imported so many different ways we have set the convention for this import to be `classnames`.